kopf/reactor/peering.py

```diff
@@ -190,7 +191,7 @@ def apply_peers(
 
             # How often do we update. Keep limited to avoid k8s api flooding.
             # Should be slightly less than the lifetime, enough for a patch request to finish.
-            await asyncio.sleep(max(1, ourselves.lifetime.total_seconds()-10))
+            await asyncio.sleep(max(1, int(ourselves.lifetime.total_seconds() - 10)))
```

Comment:
What was the problem with this? In theory, it can be float, and I believe there is nothing wrong with it (except that in practice, updating the peering object too often is not a good idea).

Reply:
just to get rid of the type checking warning of the pycharm. as  `max` function  accepts (int,int) or iterable. otherwise there was no problem with code itself.

Reply:
I see. Interesting that it wants the integers only. Anyway, the rounding does not affect the behaviour significantly.